src/v/storage/segment.h

```diff
@@ -49,8 +49,12 @@ class segment {
         /// \brief These offsets are the `batch.last_offset()` and not
         /// `batch.base_offset()` which might be confusing at first,
         /// but allow us to keep track of the actual last logical offset
+
+        // Offset of last message persisted on majority of replicas
         model::offset committed_offset;
+        // Offset of last message written to this log
         model::offset dirty_offset;
+        // Offset of last message flushed to disk in this log
```

Comment:
@dotnwat are these descriptions correct?

Reply:
they are not, log knows nothing about the replication. `committed_offset` is the one that was fsynced, `dirty_offset` is the one that was written to log (may not yet be on disk), stable offset is the one that was written do disk

Reply:
Thanks. I thought these should have comments. I will update them.